[bvl_feedback] Update TestPlan (#7028)
aces/Loris: modules/bvl_feedback/test/TestPlan.md

# Behavioural Feedback Test Plan

1. Behavioural feedback button should show up on the following pages:
 * Candidate Profile  

>  To find the Candiadate profile, first navigate to 'Access Profile', then select the specific Candidate profile from the list.  

Reviewer: This edit is not necessarily unless you'd like to change it to tell the tester _where_ to look for feedback within each of these modules/pages (as with the pencil icon).  
It is very safe to assume that anyone testing Loris knows how to get to a candidate profile and instrument list page.  I see a few inaccuracies below, i'll correct as I go ...

 ![image](images/candidate_list.PNG)  
 * Instrument List  

>  To find an instrument, first navigate to a specific Candidate Profile, then click on a specific Visit Label.  

Reviewer: ```suggestion
Feedback on a whole Visit is accessed via the Instrument List page - click the pencil icon in the blue menu bar.
```
Important - for visit-level feedback, use the pencil icon while you're looking at the instrument list module (don't click through to a visit label)

 ![image](images/visit_label.PNG)  

>  A list of Intruments shall be found in this page as shown below.  

Reviewer: ```suggestion
 A list of Instruments is found in this page as shown below.  
```
typo and changing the tone - @haoweiqiu you'll notice we've basically banned "should" and especially "shall" from all our docs - that's a conscious decision for writing style.

 ![image](images/instrument_list.PNG)  
 * Any instrument [Automation Test]   

>  *Note: The feedback button has the notepad icon in the toolbar as shown below.*  

Reviewer: across loris 
```suggestion
 *Note: The feedback button is represented by the notepad icon in the menu as shown below.*  
```
* Menu (not toolbar)

Reviewer: i think you should move this to the first line of your modifications - this pencil/notepad icon is the same trigger across multiple modules, not just in the instrument form
